setup.py

```diff
@@ -48,6 +48,7 @@
                       'numpy>=1.9.0',
                       'protobuf',
                       'six>=1.9.0'],
+    setup_requires=['Cython>=0.17'],
```

Comment:
Would you add a short comment on why Cython is required here? (required for h5py?)